lib/wor/paginate/adapters/will_paginate.rb

```diff
@@ -20,7 +20,9 @@ def count
         end
 
         def total_count
-          paginated_content.count
+          count = paginated_content.count
+          return count.size if count.is_a? Hash
```

Comment:
:thinking: I don't understand when `count` variable should be a kind of Hash class... I think count should be a number

Reply:
If AR query contains an `group_by` sometimes respond with a hash with the groups count: `{group_1: 3, group_2: 2, ..., group_n: n}`